l10n_it_account_tax_kind/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,13 @@
+**Italiano**
+
+Per configurare questo modulo, hai bisogno di:
+
+#. Andare in Fatturazione -> Configurazione -> Fiscalità italiana -> Natura esenzione imposta
+#. Creare una nuova natura esenzione imposta o modificarne una esistente
```

Comment:
Dopo il : ci andrebbe la lettera minuscola.

```suggestion
#. andare in Fatturazione -> Configurazione -> Fiscalità italiana -> Natura esenzione imposta
#. creare una nuova natura esenzione imposta o modificarne una esistente
```

Reply:
Ho notato che altri moduli di l10n-italy non hanno la minuscola in un elenco(numerato in questo caso) e/o dopo i due punti. Le faresti comunque in minuscolo?

Reply:
> Ho notato che altri moduli di l10n-italy non hanno la minuscola in un elenco(numerato in questo caso) e/o dopo i due punti. Le faresti comunque in minuscolo?

Probabilmente non sono mai stati revisionati oppure la cosa è sfuggita.
P.S.: non sono cose che decido io, è la lingua italiana che è fatta così. :smile:, 
Bisogna anche dire che nelle nostre linee guida attualmente manca la parte relativa agli elenchi puntati, vedrò di colmare questa lacuna prendendo spunto dalle altre esistenti in rete.

Per ora, in attesa dell'aggiornamento delle nostre linee guida, puoi lasciare così.

Grazie!